js/code.js

```diff
@@ -1,103 +1,60 @@
-const urlBase = 'http://159.89.91.64/LAMPAPI';
-const extension = 'php';
-
 let userId = 0;
 let firstName = "";
 let lastName = "";
 
-function doLogin()
+async function doRegister()
 {
+  userId = 0;
+  let firstName = document.getElementById("firstName").value;
+  let lastName = document.getElementById("lastName").value;
+	let login = document.getElementById("registerName").value;
+	let password = document.getElementById("registerPassword").value;
```

Comment:
Weird indentation here as well. Tabs or spaces are not a huge deal, but pick one!